docker-compose.yaml

```diff
@@ -1,24 +1,27 @@
 version: '3'
 services:
 
-  nginx-proxy:
-    image: nginx:latest
-    volumes:
-      - ./nginx.conf:/etc/nginx/nginx.conf
-      - ./certs:/etc/nginx/certs
-    ports:
-      - 443:443
```

Comment:
Are you sure you want to hog port 443? It is possible that other services may want this port. Maybe separate into a "mail" proxy and a "http" proxy?

Reply:
Sorry, you removed these lines. Please ignore.